source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionCanRecoverFromNetworkIssues.cs

```diff
@@ -239,14 +243,16 @@ public async Task WhenANetworkFailureOccurs_DuringGetCapabilities_TheClientIsAbl
                         {
                             if (capabilitiesServiceV2Exceptions.GetCapabilitiesLatestException == null)
                             {
-                                inner.EnsureTentacleIsConnectedToServer(Logger);
```

Comment:
As responses are cached, we can't make a call to the ICapabilitesService to ensure tentacle is connected but need to use another service.